pronto_utils/package.xml

```diff
@@ -1,19 +1,12 @@
 <?xml version="1.0"?>
-<package>
+<package format="3">
   <name>pronto_utils</name>
   <version>0.2.0</version>
```

Comment:
If we made a change in the API that is not backcompatible we should go for `0.3.0`, otherwise `0.2.1`. I prefer to increase the version in all of the packages (regardless of whether they are modified or not) instead of changing only the affected packages because it would be hard to understand what version matches with what version.

Reply:
There are no changes that are not backwards-compatible.

Reply:
Can you bump the version to `0.2.1` then?